pkg/loader/compose/v3.go

```diff
@@ -183,21 +183,21 @@ func dockerComposeToKomposeMapping(composeObject *types.Config) (kobject.Kompose
 		serviceConfig.CapDrop = composeServiceConfig.CapDrop
 		serviceConfig.Expose = composeServiceConfig.Expose
 		serviceConfig.Privileged = composeServiceConfig.Privileged
-		serviceConfig.Restart = composeServiceConfig.Restart
 		serviceConfig.User = composeServiceConfig.User
 		serviceConfig.Stdin = composeServiceConfig.StdinOpen
 		serviceConfig.Tty = composeServiceConfig.Tty
 		serviceConfig.TmpFs = composeServiceConfig.Tmpfs
 		serviceConfig.ContainerName = composeServiceConfig.ContainerName
 		serviceConfig.Command = composeServiceConfig.Entrypoint
 		serviceConfig.Args = composeServiceConfig.Command
-
 		// This is a bit messy since we use yaml.MemStringorInt
 		// TODO: Refactor yaml.MemStringorInt in kobject.go to int64
 		// Since Deploy.Resources.Limits does not initialize, we must check type Resources before continuing
 		if (composeServiceConfig.Deploy.Resources != types.Resources{}) {
 			serviceConfig.MemLimit = libcomposeyaml.MemStringorInt(composeServiceConfig.Deploy.Resources.Limits.MemoryBytes)
 		}
+		//Handling restart-policy
+		serviceConfig.Restart = composeServiceConfig.Deploy.RestartPolicy.Condition
```

Comment:
Please add a small comment here in regards to deploy.

Example:

```
// Here we handle all Docker Compose Deploy keys
```

So we can better organize from the rest of all the keys :)